src/commands/refocus-collector-pause.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * src/commands/refocus-collector-pause.js
+ *
+ * Executes the "pause" command.
+ */
+'use strict'; // eslint-disable-line strict
+const program = require('commander');
+const logger = require('winston');
+const cmdUtils = require('./utils');
+const doPost = require('../utils/httpUtils.js').doPostToRefocus;
+
+program
+  .option('-n, --collectorName <collectorName>', 'The name of the ' +
+    'collector to be paused')
+  .option('-u, --refocusUrl <refocusUrl>', 'The url of the refocus ' +
+    'instance this collector will send to')
+  .option('-t, --accessToken <accessToken>', 'A valid refocus token')
+  .option('-r, --refocusProxy <refocusProxy>', 'Proxy to Refocus')
+  .parse(process.argv);
+
+if (!cmdUtils.validateArgs(program)) {
+  process.exit(1);
+}
+
+const config = cmdUtils.setupConfig(program);
+const pausePath = `/v1/collectors/${config.name}/pause`;
+
+logger.log('Pause =>', config.name, config.refocus.url + pausePath);
+
+// Request to Refocus to pause the collector
+doPost(pausePath)
+.then(() => {
+  logger.info(`Request to pause ${config.name} has been accepted.`);
```

Comment:
maybe the message should just be "Pausing [name]. Use [command] or [API endpoint] to resume."
what do you think?

Reply:
I used the word`accepted` because we are not really pausing until the heartbeat is sent back to the collector.  I am fine either ways. Let me know your preference.